src/functions.c

```diff
@@ -406,7 +459,14 @@ void functionStatsCommand(client *c) {
     dictEntry *entry = NULL;
     while ((entry = dictNext(iter))) {
         engineInfo *ei = dictGetVal(entry);
+        addReplyMapLen(c, 3);
+        addReplyBulkCString(c, "name");
         addReplyBulkCString(c, ei->name);
```

Comment:
maybe the engine list should be a map? instead of an array with an map entry named "name"?

Reply:
So map of maps?

Reply:
yes, rather than an array of maps (and an entry named "name").
don't you think that's better?

this is actually similar to the difference between the old COMMAND INFO and the new COMMAND DOCS.
one is an array of arrays, and the other is a map or maps.

Reply:
thanks.
looks better IMHO.
can you please also update the doc pr?

Reply:
Updated